numpyro/distributions/continuous.py

```diff
@@ -1680,38 +1680,46 @@ def infer_shapes(df, loc, scale_tril):
         return batch_shape, event_shape
 
 
-class MultivariateBeta(Distribution):
+class GaussianCopula(Distribution):
     arg_constraints = {
-        "concentration1": constraints.positive,
-        "concentration0": constraints.positive,
+        "covariance_matrix": constraints.positive_definite,
+        "precision_matrix": constraints.positive_definite,
+        "scale_tril": constraints.lower_cholesky,
     }
-    reparametrized_params = ["concentration1", "concentration0"]
+    support = constraints.real_vector
+    reparametrized_params = [
+        "covariance_matrix",
+        "precision_matrix",
+        "scale_tril",
```

Comment:
I think we only need to use correlation matrix to account for the correlation of the variables here (so only `correlation_matrix`/`correlation_cholesky` are needed). Otherwise, we might face identification issue where multiple `covariance_matrix` parameters give the same distribution.

Reply:
Just to make sure I understand, is the suggestion to keep `scale_tril`? I'm not quite sure how to keep the correlations only (since the individual variances are not important, as you mentioned)